### PR TITLE
bababooey

### PR DESCRIPTION
**Pull request by NopemanMcHalt**
## About The Pull Request
bababooey

## Why It's Good For The Game
bababooey

## Changelog
:cl: bababooey
add: bababooey
/:cl:

## P.S.
~~I regret nothing~~ bababooey

### CODE REVIEW

**Comment by wesoda25** [comment]
fascinating

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -503,3 +503,102 @@
...
+/datum/emote/living/bababooey/can_run_emote(mob/living/user, status_check = TRUE , intentional)
+	. = ..()
+	if(. && iscarbon(user))
+		var/mob/living/carbon/C = user
+		return !C.silent
+
+/datum/emote/living/bababooey/get_sound(mob/living/user)
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.dna.species.id == "human" && (!H.mind || !H.mind.miming))
```
Comment (ATH1909) [inline code/modules/mob/living/emote.dm:524]: let moths bababooey you monster

Comment (ATH1909) [reply code/modules/mob/living/emote.dm:524]: let borgs do it too

Comment (Fikou) [reply code/modules/mob/living/emote.dm:524]: ath is a moth player and a borg main do not trust their words

**Comment by MrDoomBringer** [comment]
simply epic

**Comment by ATH1909** [comment]
speedmerge

File: code/modules/mob/living/emote.dm (hunk 1 above)
Comment (ATH1909) [reply code/modules/mob/living/emote.dm:524]: I just wanna bababooey, man

Comment (NopemanMcHalt) [reply code/modules/mob/living/emote.dm:524]: Initially planned to be human and subspecies only (I'm kind of retarded so initial commit just used modified laugh emote w/out changing much to account species code), but will make it for borgs too for additional bababooey points.

**Comment by optimumtact** [comment]
I don't get it but the voices made me laugh.

**Comment by TheSpaghetti** [comment]
merge this or no balls

**Comment by TheSpaghetti** [comment]
bababooey

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -503,3 +503,89 @@
...
+		if(!L.mind || !L.mind.miming)
+			if(user.is_muzzled())
+				return 'sound/voice/human/ffff.ogg'
+			else
+				return 'sound/voice/human/hohohoy.ogg'
+
+/datum/emote/living/ffff
+	key = "ffff"
+	key_third_person = "ffffs"
+	message = "spews something untelligble."
```
Comment (tralezab) [inline code/modules/mob/living/emote.dm:582]: untelligble > unintelligible

**Comment by MMMiracles** [comment]
https://www.youtube.com/watch?v=8Scm09bwT_s

**Comment by TheVekter** [comment]
WHAT IS THIS, THE FUCKIN' NINETIES

**Review by Cyberboss — APPROVED** [review]

**Comment by ATH1909** [comment]
holy fuck based

**Comment by Qustinnus** [comment]
why cyberboss

**Comment by guyman500** [comment]
bababooey

**Comment by MrDoomBringer** [comment]
fa fa foohey

**Comment by TheVekter** [comment]
holy shit he actually did it, what a madman

fafafogey

**Comment by MortimerMcMire** [comment]
never remove this

add a 20 second cooldown or whatever and take out the ones besides the classic bababooey

**Pull request merged**